src/bananas/environment.py

```diff
@@ -272,7 +272,7 @@ def get_bool(self, key: str, default: U) -> Union[bool, U]:
     def get_bool(self, key: str, default: None = None) -> Optional[bool]:
         ...
 
-    def get_bool(self, key: str, default: object = None) -> object:
```

Comment:
This was probably unintentional, right? :)

Reply:
Yep!